contrib/blueflood-docker/configs/ES-Setup/init-es.sh

```diff
@@ -0,0 +1,79 @@
+#!/bin/bash
```

Comment:
Is this copied from blueflood-elasticsearch/src/main/resource/init-es.sh?  Same copy questions as before.